### ISSUE TITLE
`commandflow-discord` should use Application commands

### ISSUE THREAD

**Issue by yusshu** [unnamed/command-flow#32]
`commandflow-discord` currently uses messages to detect commands. We should use [Application Commands](https://discord.com/developers/docs/interactions/application-commands) instead since it's a more user-friendly and native way to create bot commands.

Also, bots can't read message contents without a privileged intent

**Comment by FixedDev** [comment]
Yeah, it should be done that way, it’s only programmed like that since the module predates discord application commands, but this should be worked on
